Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1770,7 +1770,7 @@ def fetch_incidents():
 
 def threat_to_incident(threat):
     incident = {}
-    incident['name'] = 'Sentinel One Threat: ' + threat.get('classification')
+    incident['name'] = 'Sentinel One Threat: ' + str(threat.get('classification', 'Not classified'))
```

Comment:
can you please check if there are any other params we can put instead here? maybe threat name or something of this sort?